doc/changelogs/CHANGELOG_V19.md

```diff
@@ -38,7 +39,151 @@
+* \[[`d08a574ecf`](https://github.com/nodejs/node/commit/d08a574ecf)] - **tls**: fix re-entrancy issue with TLS close\_notify (David Benjamin) [#44563](https://github.com/nodejs/node/pull/44563)
+* \[[`e53cfae20e`](https://github.com/nodejs/node/commit/e53cfae20e)] - **tools**: update lint-md-dependencies to rollup\@3.9.0 (Node.js GitHub Bot) [#46039](https://github.com/nodejs/node/pull/46039)
+* \[[`31d22c1dca`](https://github.com/nodejs/node/commit/31d22c1dca)] - **tools**: update doc to unist-util-select\@4.0.2 (Node.js GitHub Bot) [#46038](https://github.com/nodejs/node/pull/46038)
+* \[[`59666358d8`](https://github.com/nodejs/node/commit/59666358d8)] - **tools**: add release host var to promotion script (Ruy Adorno) [#45913](https://github.com/nodejs/node/pull/45913)
+* \[[`669cfad0a7`](https://github.com/nodejs/node/commit/669cfad0a7)] - **tools**: add url to `AUTHORS` update automation (Antoine du Hamel) [#45971](https://github.com/nodejs/node/pull/45971)
+* \[[`4120739d34`](https://github.com/nodejs/node/commit/4120739d34)] - **tools**: update lint-md-dependencies to rollup\@3.8.1 (Node.js GitHub Bot) [#45967](https://github.com/nodejs/node/pull/45967)
+* \[[`fb0a7e5b79`](https://github.com/nodejs/node/commit/fb0a7e5b79)] - **tools**: update GitHub workflow action (Mohammed Keyvanzadeh) [#45937](https://github.com/nodejs/node/pull/45937)
+* \[[`54332fce84`](https://github.com/nodejs/node/commit/54332fce84)] - **tools**: update lint-md dependencies (Node.js GitHub Bot) [#45813](https://github.com/nodejs/node/pull/45813)
+* \[[`070a5f7c8f`](https://github.com/nodejs/node/commit/070a5f7c8f)] - **tools**: enforce use of trailing commas in `tools/` (Antoine du Hamel) [#45889](https://github.com/nodejs/node/pull/45889)
+* \[[`ff14282ee8`](https://github.com/nodejs/node/commit/ff14282ee8)] - **tools**: add `ArrayPrototypeConcat` to the list of primordials to avoid (Antoine du Hamel) [#44445](https://github.com/nodejs/node/pull/44445)
+* \[[`124c2b32d9`](https://github.com/nodejs/node/commit/124c2b32d9)] - **tools**: fix incorrect version history order (Fabien Michel) [#45728](https://github.com/nodejs/node/pull/45728)
+* \[[`27cf389c22`](https://github.com/nodejs/node/commit/27cf389c22)] - **tools**: update eslint to 8.29.0 (Node.js GitHub Bot) [#45733](https://github.com/nodejs/node/pull/45733)
+* \[[`8ee1d5dee5`](https://github.com/nodejs/node/commit/8ee1d5dee5)] - **util**: add fast path for text-decoder fatal flag (Yagiz Nizipli) [#45803](https://github.com/nodejs/node/pull/45803)
+* \[[`6f1a1802c3`](https://github.com/nodejs/node/commit/6f1a1802c3)] - **vm**: refactor to use `validateStringArray` (Deokjin Kim) [#46020](https://github.com/nodejs/node/pull/46020)
+* \[[`7bd6a2c258`](https://github.com/nodejs/node/commit/7bd6a2c258)] - **wasi**: fast calls (snek) [#43697](https://github.com/nodejs/node/pull/43697)
 
 ## 2022-12-14, Version 19.3.0 (Current), @targos
```

Comment:
The anchor for 19.3.0 (`<a id="19.3.0"></a>`) shouldn't be removed.

Reply:
That's weird. It seems caused by `make format-md`. Fixed.